jdi-light/src/main/java/com/epam/jdi/light/settings/JDISettings.java

```diff
@@ -1,11 +1,28 @@
 package com.epam.jdi.light.settings;
 
 public class JDISettings {
-    public static CommonSettings COMMON = new CommonSettings();
-    public static LogSettings LOGS = new LogSettings();
-    public static Timeouts TIMEOUTS = new Timeouts();
-    public static DriverSettings DRIVER = new DriverSettings();
-    public static ElementSettings ELEMENT = new ElementSettings();
-    public static PageSettings PAGE = new PageSettings();
-    public static ScreenSettings SCREEN = new ScreenSettings();
+    private JDISettings() {
+        LOGS = new LogSettings();
+        TIMEOUTS = new Timeouts();
+        ELEMENT = new ElementSettings();
+        PAGE = new PageSettings();
+        SCREEN = new ScreenSettings();
+        DRIVER = new DriverSettings();
+    }
+
+    private static JDISettings jdiSettings;
+
+    public static JDISettings getJDISettings() {
+        if (jdiSettings == null) {
+            jdiSettings = new JDISettings();
+        }
+        return jdiSettings;
+    }
+
+    public LogSettings LOGS;
+    public Timeouts TIMEOUTS;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)